.github/workflows/build.yml

```
@@ -93,28 +94,33 @@ jobs:
           name: kwctl-linux-${{ matrix.targetarch }}
           path: kwctl-linux-${{ matrix.targetarch }}.zip
 
-      - name: Install SBOM generator tool
-        uses: kubewarden/github-actions/sbom-generator-installer@d849020c9137340c2373d1cbc9cc571b2b18c17e # v2
+      - name: Install the syft command
+        uses: kubewarden/github-actions/syft-installer@2b1605e00a16f618214dbfff64e020f07e4a4ecb # v3.1.6
 
-      - name: Generate SBOM
+      - name: Create SBOM file
+        shell: bash
         run: |
-          spdx-sbom-generator -f json
-          # SBOM files should have "sbom" in the name due the CLO monitor
-          # https://clomonitor.io/docs/topics/checks/#software-bill-of-materials-sbom
-          mv bom-cargo.json kwctl-linux-${{ matrix.targetarch }}-sbom.spdx.json
-
-      - name: Sign BOM file
+          syft \
+          --file kwctl-linux-${{ matrix.targetarch }}-sbom.spdx \
+          --output spdx-json \
+          --source-name kwctl-linux-${{ matrix.targetarch }} \
+          --source-version ${{ github.sha }} \
+          -vv \
+          dir:. # use dir default catalogers, which includes Cargo.toml
+
+      - name: Sign SBOM file
```

I think you didn't add the shell directive, which would cause an error to happen
```suggestion

      - name: Sign SBOM file
        shell: bash
```